include/bartender.h

```diff
@@ -137,22 +140,24 @@ class Bartender
     {
         // 9, 10 do inicjalizacji - op[isac w komentarzu jako SD2 i SD3
         // servo_.write(servo_angle_down);
-        digitalWrite(servoPullDownPin, HIGH);
-        digitalWrite(servoPullUpPin, LOW);
+        analogWrite(SRV_PWM_PIN, 128);
+        digitalWrite(SRV_DOWN_PIN, HIGH);
+        digitalWrite(SRV_UP_PIN, LOW);
         delay(pullupTimeMs);
 
-        digitalWrite(servoPullDownPin, LOW);
-        digitalWrite(servoPullUpPin, LOW);
+        digitalWrite(SRV_DOWN_PIN, LOW);
+        digitalWrite(SRV_UP_PIN, LOW);
     }
 
     void PullShaker()
     {
-        digitalWrite(servoPullDownPin, LOW);
-        digitalWrite(servoPullUpPin, HIGH);
+        analogWrite(SRV_PWM_PIN, 128);
```

Comment:
You can consider splitting logic of this function into 2 separate ones. In clean code rules it is stated that function should have only 1 responsibility. This one controls pulling up and down and control of mixing.